.gitignore

```diff
@@ -1,3 +1,12 @@
 node_modules
-test/data/babylon
+test/data/babel-parser
 test/data/typescript-compiler
+
+# Ignore TypeScript-emitted files
+*.js
+*.js.map
+*.d.ts
+*.d.ts.map
+# Except...
+!/types/**/*.d.ts
+!babel-register.js
```

Comment:
Didn't see a good way around this, given that we wouldn't want to change the import paths. Open to ideas

Reply:
I adopted your strategy from https://github.com/benjamn/recast/pull/554#issuecomment-440975441 of just using `tsc` and https://www.npmjs.com/package/ts-add-module-exports, so at least the `!babel-register.js` part of this is resolved (since we're not using Babel anymore).